index.html

```diff
@@ -41,32 +49,40 @@ <h2 id="description-text">I'm a Software Developer</h2>
                          
                 </section>
             </section>  
-        </article> 
+            
+        
+    </article> 
         <article id="work-section">
                 <section class="recent-work">
-                    <h2 id="recent-work-title">My Recent Works</h2>
-                    <hr>
+                    <div class="title-recent-word"><h2 id="recent-work-title">My Recent Works</h2>
+                    <hr></div>
                     <img id="img-project" src="images/Img-Placeholder.png" alt="images-work">
+                    <div class="multipost">
+                    <div class="big-image"></div>
+                    <div>
                     <h3 id="project-description-title">Multi-Post Stories</h3>
                     <p id="project-description">A daily selection of privately personalized reads;
                        no accounts or sign-ups required. has been the industry's standard dummy
                        text ever since the 1500s, when an unknown printer took a standard dummy text.
                     </p>
                     <section class="button-section">
```

Comment:
- [ ] You are doing a great job so far but I think you can still improve this section better to closely look like the Figma design. 💯 
- - On screen sizes between `764px` and `1130px` the content of your recent-work overflows the width of the screen, please consider fixing this as good and responsive UI is paramount to the overall user experience of your website. 😄 👍 
- Screenshot:
![Screenshot 2023-04-01 094455](https://user-images.githubusercontent.com/107632780/229276983-d89e0684-2078-4694-9ab7-9caed05cf4fc.png)